core/src/cg/group4/view/screen/CollectiblesScreen.java

```diff
@@ -0,0 +1,198 @@
+ */
+public class CollectiblesScreen extends ScreenLogic {
+	
+	/**
+	 * cContentTable contains the collectibles of the collection.
+	 * cContainer contains the scrollpane displaying the collectibles.
+	 */
+	protected Table cContentTable, cContainer;
+	
+	/**
+	 * ScrollPane displaying the collectibles of the collection.
+	 */
+	protected ScrollPane cScrollPane;
+	
+	/**
+	 * Collection of the player.
+	 */
+	protected Collection cCollection;
+	
+	/**
+	 * Button that takes the player back to the home screen.
+	 */
+	protected TextButton cBackButton;
+	
+	/**
+	 * Numbers of collectibles displayed without scrolling, and the amount of
+	 * checkboxes in the menu.
+	 */
+	protected final int cItemsOnScreen = 10, cNumberOfTopBarItems = 3;
+	
+	/**
+	 * Object that creates images for the collectibles.
+	 */
+	protected CollectibleDrawer cDrawer;
+	
+	/**
+	 * Object that sorts the collectibles in the collection.
+	 */
+	protected CollectibleSorter cSorter;
+	
+	/**
+	 * Checkbox that sets the CollectibleSorter to sorting by rarity.
+	 */
+	protected CheckBox cSortRarity;
+	
+	/**
+	 * Selectbox that contains the groups that the user is currently in.
+	 */
+	protected SelectBox<String> cGroupsBox;
+	
+	/**
+	 * Selectbox that contains all the possible sorting options.
+	 */
+	protected SelectBox<String> cSortBox;
+	
+	/**
+	 * Creates a new CollectibleScreen.
+	 * @param collection of the player.
+	 */
+	public CollectiblesScreen(final Collection collection) {
+		cCollection = collection;
+		cDrawer = new CollectibleDrawer();
+		cSorter = new SortByRarity();
+	}
+
+	@Override
+	protected WidgetGroup createWidgetGroup() {
+		final int screenWidth = Gdx.graphics.getWidth();
+		final int screenHeight = Gdx.graphics.getHeight();
+		cDrawer = new CollectibleDrawer();
+		
+		cContainer = new Table();
+		cContainer.setFillParent(true);
+		
+		cBackButton = cGameSkin.generateDefaultMenuButton("Back");
+		cBackButton.addListener(new ChangeListener() {
+            @Override
+            public void changed(final ChangeEvent event, final Actor actor) {
+                ScreenStore.getInstance().setScreen("Home");
+            }
+        });
+				
+		cContentTable = new Table();
+		cContentTable.setWidth(screenWidth);
+		cScrollPane = new ScrollPane(cContentTable);
+		cScrollPane.setForceScroll(false, true);
+		
+		cSortBox = cGameSkin.generateDefaultSelectbox();
+		String[] def = new String[2];
+		def[0] = "Sort_Rarity";
+		def[1] = "<Insert Sort Object>";
+		cSortBox.setItems(def);
+		cSortBox.addListener(new ChangeListener() {
+			@Override
+			public void changed(final ChangeEvent event, final Actor actor) {
+				System.out.println("Selected Sorting: " + cSortBox.getSelected());
+			}
+		});
+		
+		cGroupsBox = cGameSkin.generateDefaultSelectbox();
+		String[] abc = new String[3];
+		abc[0] = "My Collection";
+		abc[1] = "Group_1";
+		abc[2] = "Group_2";
+		cGroupsBox.setItems(abc);
```

Comment:
Personally I prefer the current code. 
It is visible what it does with at a single glance.
However since the amount of Strings is low enough (3), I would accept both.